tapchannel/aux_funding_controller.go

```diff
@@ -1028,13 +1031,6 @@ func (f *FundingController) completeChannelFunding(ctx context.Context,
 		return nil, err
 	}
 
-	fundingState.lockedAssetInputs = fn.Map(
```

Comment:
I moved this down so we track the locked inputs as soon as the initial vPSBT funding finishes.